src/core/reference/include/openvino/reference/interpolate.hpp

```diff
@@ -408,24 +408,34 @@ void InterpolateEval<T>::linear_onnx_func(const T* input_data, T* out) {
         ((input_rank == 2) && (num_of_axes == 2) && (m_axes[0] == 0) && (m_axes[1] == 1)) ||
         ((input_rank == 3) && (num_of_axes == 3) && (m_axes[0] == 0) && (m_axes[1] == 1) && (m_axes[2] == 2));
 
+    bool channel_last = false;
+
     if (input_rank >= 4) {
         std::vector<int64_t> all_axes;
         std::vector<int64_t> axes_without_batch_and_channels;
+        std::vector<int64_t> axes_without_batch_and_channels_last;
         all_axes.push_back(0);
         all_axes.push_back(1);
+        axes_without_batch_and_channels_last.push_back(1);
         for (int64_t i = 2; i < static_cast<int64_t>(input_rank); ++i) {
             all_axes.push_back(i);
             axes_without_batch_and_channels.push_back(i);
+            if (i != static_cast<int64_t>(input_rank) - 1)
+                axes_without_batch_and_channels_last.push_back(i);
         }
 
         correct_axes = ((num_of_axes == input_rank) && (m_axes == all_axes)) ||
-                       ((num_of_axes == input_rank - 2) && (m_axes == axes_without_batch_and_channels));
+                       ((num_of_axes == input_rank - 2) && (m_axes == axes_without_batch_and_channels)) ||
+                       ((num_of_axes == input_rank - 2) && (m_axes == axes_without_batch_and_channels_last));
+
+        if ((num_of_axes == input_rank - 2) && (m_axes == axes_without_batch_and_channels_last))
+            channel_last = true;
```

Comment:
Could you please check and share how the Interpolate layer in the failing conformance test looks like, what are the `axes` values and input shapes of the Interpolate in the model from the test mentioned in the issue (https://github.com/openvinotoolkit/openvino/issues/23553).

Then please add corresponding unit test for such cases here:
https://github.com/openvinotoolkit/openvino/blob/a5cf8b30d5430a94e3b62f77700d8b904cc1b592/src/plugins/template/tests/functional/op_reference/interpolate.cpp#L564

Reply:
> Could you please check and share how the Interpolate layer in the failing conformance test looks like, what are the `axes` values and input shapes of the Interpolate in the model from the test mentioned in the issue (#23553).
> 
> Then please add corresponding unit test for such cases here:
> 
> https://github.com/openvinotoolkit/openvino/blob/a5cf8b30d5430a94e3b62f77700d8b904cc1b592/src/plugins/template/tests/functional/op_reference/interpolate.cpp#L564

Below is the failed conformance case. If I understood it correctly, it put the batch_number = 64 at its last dimension, and spatial dimensions are in the middle two.

![image](https://github.com/openvinotoolkit/openvino/assets/50537048/c2f73b04-acd0-4207-9cf9-20928e47e4d7)